cmd/operator/README.md

```diff
@@ -135,6 +137,10 @@ command line argument multiple times.
 
 ## Known issues
 
+### Multiple Custom Resources
+
+With multiple custom resources `nodeSelector` has to be carefully set to avoid scenarios where the device plugin daemonset gets deployed twice on same node. The operator does not cross-check `nodeSelector`s nor Pod scheduling to nodes. Multiple plugins on same node could cause invalid behaviour and/or duplicate device resources on node.
```

Comment:
Didn't really understand the second clause.  Maybe it would be clearer like this?
```suggestion
With multiple custom resources, `nodeSelector` has to be carefully set to avoid device plugin DaemonSet getting deployed multiple times on the same node, as operator does not check or prevent this. Multiple plugins managing same resource on a node can cause invalid behavior and/or duplicate device resources on node.
```

Reply:
+1 to @eero-t 's suggestion. The problem really is the "same resource on a node". In theory two plugin instances of the same device type *can* run on the same node if they monitor different resource types but this is not really useful in real scenarios.

Reply:
Sure. I'll update the wording.